pandas/tests/series/test_repr.py

```diff
@@ -487,3 +487,23 @@ def test_categorical_series_repr_timedelta_ordered(self):
                                    8 days 01:00:00 < 9 days 01:00:00]"""  # noqa
 
         assert repr(s) == exp
+
+    def test_info(self, capsys):
```

Comment:
Looks like there are a lot of tests that exists for frame that don't exist here; can you try to parametrize them or at least re-use them here? (not all may be applicable)